doc/articles/toc.yml

```diff
@@ -428,6 +428,8 @@
           href: uno-development\creating-runtime-tests.md
         - name: UI tests with Uno.UITest
           href: uno-development\creating-ui-tests.md
+    - name: Debugging Solution Templates
+      href: uno-development\debugging-templates.md
     - name: Conventional Commits format
       href: uno-development\git-conventional-commits.md
```

Comment:
Is there a logical ordering in this file?

Reply:
it's grouped into topics, and roughly ordered by importance within each topic (or alphabetically for a few topics eg WinRT features)